universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -103,7 +103,22 @@ private ContainerTag buildQuestionContainer(String title) {
             Styles.OVERFLOW_Y_AUTO,
             Styles.RELATIVE,
             Styles.W_2_5)
-        .with(renderHeader(title, Styles.CAPITALIZE));
+        .with(renderHeader(title, Styles.CAPITALIZE))
+        .with(multiOptionQuestionField());
+  }
+
+  // A hidden template for multi-option questions.
+  private ContainerTag multiOptionQuestionField() {
+    return div()
+        .with(
+            FieldWithLabel.input()
+                .setFieldName("option")
+                .setLabelText("Question option")
+                .getContainer()
+                .withClasses(Styles.FLEX, Styles.ML_2),
+            button("Remove").withType("button").withClasses(Styles.FLEX, Styles.ML_4))
+        .withId("multi-option-question-answer-template")
+        .withClasses(Styles.HIDDEN, Styles.FLEX, Styles.FLEX_ROW, Styles.MB_4);
```

Comment:
Nit. Consider defining the input and remove button outside of the return statement for more readable code...

```suggestion
    ContainerTag optionInput = FieldWithLabel.input()
                .setFieldName("option")
                .setLabelText("Question option")
                .getContainer()
                .withClasses(Styles.FLEX, Styles.ML_2);
    ContainerTag removeOptionButton = 
            button("Remove").withType("button").withClasses(Styles.FLEX, Styles.ML_4);
    return div(optionInput, removeOptionButton)
        .withId("multi-option-question-answer-template")
        .withClasses(Styles.HIDDEN, Styles.FLEX, Styles.FLEX_ROW, Styles.MB_4);
```
or


```suggestion
    ContainerTag optionInput = FieldWithLabel.input()
                .setFieldName("option")
                .setLabelText("Question option")
                .getContainer()
                .withClasses(Styles.FLEX, Styles.ML_2);
    ContainerTag removeOptionButton = 
            button("Remove").withType("button").withClasses(Styles.FLEX, Styles.ML_4);
    return div()
        .withId("multi-option-question-answer-template")
        .withClasses(Styles.HIDDEN, Styles.FLEX, Styles.FLEX_ROW, Styles.MB_4)
        .with(optionInput, optionButton);
```

Reply:
Much more readable, thank you!